support-frontend/assets/helpers/tracking/googleTagManager.js

```diff
@@ -30,8 +29,26 @@ type GaEventData = {
   label: ?string,
 }
 
+const googleAnalyticsKey = 'googleAnalytics';
+const googleTagManagerKey = 'googleTagManager';
+
+const sourcePointVendorIds: {
```

Comment:
it might be good not to mention sourcepoint, since that's really an implementation detail of the CMP and may change. i'm not sure if the vendor ids would change then?

Reply:
Yes: these `vendorIds` are specific to Sourcepoint's implementation.